src/layout/boxmodel.ts

```diff
@@ -50,9 +50,11 @@ export function updates(): BoxModelData[] {
 
 export function relative(x: number, y: number, element: Element): number[] {
     if (x && x >= 0 && y && y >= 0 && element) {
-        let box = layout(element);
+        let scrollX = "pageXOffset" in window ? window.pageXOffset : document.documentElement.scrollLeft;
+        let scrollY = "pageYOffset" in window ? window.pageYOffset : document.documentElement.scrollTop;
+        let box = layout(element, scrollX, scrollY);
```

Comment:
Use de-structuring assignment to write

let [x, y, width, height] = layout(....)

Rather than box[0] as here.

Reply:
Done, looks much cleaner. Thanks.